internal/distro/distro.go

```diff
@@ -51,6 +51,10 @@ var (
 	swapMkfsCmd  = "/usr/sbin/mkswap"
 	vfatMkfsCmd  = "/usr/sbin/mkfs.vfat"
 	xfsMkfsCmd   = "/usr/sbin/mkfs.xfs"
+
+	// Needed for blackbox tests
+	losetupCmd = "/sbin/losetup"
+	mke2fsCmd  = "/sbin/mke2fs"
```

Comment:
The other paths default to `/usr/sbin/...`, any reason to not use the same path here?